examples/integration_test.py

```diff
@@ -1217,6 +1325,10 @@ def print_usage(exitcode, reason=None):
         print('=' * 30, 'Verifying Admin API', '=' * 30)
         verify_admin()
 
+    if 'avro-https' in modes:
+        print('=' * 30, 'Verifying AVRO with https', '=' * 30)
```

Comment:
HTTPS

Reply:
I humbly disagree but I have made the change anyway